snapcraft/commands/validation_sets.py

```diff
@@ -60,10 +60,10 @@
 
 
 class StoreEditValidationSetsCommand(BaseCommand):
-    """Command for edit-validation-sets."""
+    """Edit a validation set.."""
```

Comment:
I wish I noticed this prior to all the tests passing!

Can you remove the extra period here?